Cilsil/Log.cs

```diff
@@ -133,7 +133,13 @@ public static void Write(string s, ConsoleColor c)
         /// <summary>
         /// TODO: use https://nlog-project.org or log4net instead of this class.
         /// </summary>
-        public static void WriteError(string s) => WriteLine(s, ConsoleColor.Red);
+        public static void WriteError(string s) 
+        {
+            if (debugMode)
```

Comment:
an actual error should not be contingent on it being debug mode, as an error should actually be reported.

Reply:
Changed this back, and instead changed error logs in translation to warning logs